DataConnectors/ForgeRock_CEF.json

```diff
@@ -0,0 +1,112 @@
+      "type" : "IsConnectedQuery",
+      "value" : [
+        "\nCommonSecurityLog\n| where DeviceVendor == \"PROVIDER NAME\"\n| where DeviceProduct == \"APPLIANCE NAME\"\n\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+      ]
+    }
+  ],
+  "availability" : {
+    "status" : 1
+  },
+  "permissions" : {
+    "resourceProvider" : [
+      {
+        "provider" : "Microsoft.OperationalInsights/workspaces",
+        "permissionsDisplayText" : "read and write permissions are required.",
+        "providerDisplayName" : "Workspace",
+        "scope" : "Workspace",
+        "requiredPermissions" : {
+          "read" : true,
+          "write" : true,
+          "delete" : true
+        }
+      }
+    ]
+  },
+  "instructionSteps" : [
+    {
+      "title" : "Configuration for the ForgeRock Common Audit (CAUD) for Microsoft Azure Sentinel",
+      "description" : "In ForgeRock, install and configure this Common Audit (CAUD) for Azure Sentinel per the documentation at https://github.com/javaservlets/SentinelAuditEventHandler. Next, in Azure, follow the below CEF steps."
+    },
+    {
+      "title" : "\n\n\n1. Linux Syslog agent configuration",
+      "description" : "Install and configure the Linux agent to collect your Common Event Format (CEF) Syslog messages and forward them to Azure Sentinel.\n\n> Notice that the data from all regions will be stored in the selected workspace",
+      "innerSteps" : [
+        {
+          "title" : "1.1 Select or create a Linux machine",
+          "description" : "Select or create a Linux machine that Azure Sentinel will use as the proxy between your security solution and Azure Sentinel this machine can be on your on-prem environment, Azure or other clouds."
+        },
+        {
+          "title" : "1.2 Install the CEF collector on the Linux machine",
+          "description" : "Install the Microsoft Monitoring Agent on your Linux machine and configure the machine to listen on the necessary port and forward messages to your Azure Sentinel workspace. The CEF collector colects CEF messages on port 514 TCP.\n\n> 1. Make sure that you have Python on your machine using the following command: python –version.\n\n> 2. You must have elevated permissions (sudo) on your machine.",
+          "instructions" : [
+            {
+              "parameters" : {
+                "fillWith" : [
+                  "WorkspaceId",
+                  "PrimaryKey"
+                ],
+                "label" : "Run the following command to install and apply the CEF collector:",
+                "value" : "sudo wget https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_installer.py&&sudo python cef_installer.py {0} {1}"
+              },
+              "type" : "CopyableLabel"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "title" : "2. Forward Common Event Format (CEF) logs to Syslog agent",
```

Comment:
I see weird characters in step 2 after "port 514 TCP on the machineâs IP address." Please delete the text "machine's IP" and retype to ensure it renders fine. You should be able to validate and see if this is fixed by the test utility https://portal.azure.com/?feature.BringYourOwnConnector=true per the data connector guidance. Please test and fix these.

Reply:
I don't see those characters... could be a MacOS thing?

Reply:
Ie, I always modify the json, upload to ?feature.BringYourOwnConnector first, verify sencond, and only then copy and paste into github. somewhere in that last step things are weird; and note those are the OOTB microsoft instructions I am copying and pasting

Reply:
I can still see the ASCII chars in both places. This PR has been in progress for a long time. I'll approve and I'll make the make the change